### ISSUE TITLE
Cannot complete with keyboard navigation

### ISSUE THREAD

**Issue by Sigmanificient** [NixOS/nixos-search#1408]
The current autocomplete selector closes when focusing it!

<img width="1215" height="533" alt="Image" src="https://github.com/user-attachments/assets/d87c19b7-6ae8-4859-a92e-90330a874c40" />

**Comment by KiaraGrouwstra** [comment]
now i wonder if this didn't like work before - i tend to use keyboard myself
